R/read_csv.R

```diff
@@ -225,27 +226,27 @@ read_cmdstan_csv <- function(files,
           if (length(variables) > 0) {
             warmup_draws <- posterior::bind_draws(
               warmup_draws,
-              posterior::as_draws_array(draws[1:num_warmup_draws, variables]),
+              posterior::as_draws_array(draws[1:num_warmup_draws, variables, drop = FALSE]),
```

Comment:
Yeah I forgot that we need drop=FALSE because posterior handles that automatically. But in this case we're indexing into `draws` _before_ converting it to a posterior draws array object.